kafka-streams/src/main/java/StreamsFilterBalance2.java

```diff
@@ -14,11 +14,13 @@
 import java.io.IOException;
 import java.text.SimpleDateFormat;
 import java.util.*;
+import java.util.stream.Collectors;
 
 public class StreamsFilterBalance2 {
 
     private static JsonParser jsonParser = new JsonParser();
     private static Map<String, Account> accounts = new HashMap<>();
+    private static Map<String, Account> last100accs = new HashMap<>();
```

Comment:
where are you using last100accs?

Reply:
You are right, last100accs is not being used. Initially, I thought I should save the entire payload for 100 record window, but then I figured that just saving balance amount from the latest 100 records is enough for the purpose. So, i ended up not using this variable. I'll remove it and send another pull request.